Automattic/wp-calypso
# Move mshots functions from my-sites to lib

#### Changes proposed in this Pull Request

This PR moves the mshots related functions, currently found in `site-importer/site-preview-actions.js`, to the more generic `lib/mshots` file, so that we can start using these functions elsewhere. This will be useful for #31721 as it'll allow us to avoid crossing modules to use them.

#### Testing instructions

- Go to http://calypso.localhost:3000/settings/import/
- Go through the Wix flow until you hit the preview site step.
- You should see the preview as expected.
